src/Model.ts

```diff
@@ -1,41 +1,31 @@
-// @flow
-import {
-  observable,
-  action,
-  ObservableMap,
-  computed,
-  toJS,
-  runInAction
-} from 'mobx'
-import { uniqueId, union, isEqual, isPlainObject } from 'lodash'
-import deepmerge from 'deepmerge'
-import Collection from './Collection'
-import apiClient from './apiClient'
+import { ObservableMap, action, computed, observable, runInAction, toJS } from 'mobx'
+import { debounce, includes, isEqual, isPlainObject, uniqueId, union } from 'lodash'
 import Base from './Base'
+import Collection from './Collection'
 import Request from './Request'
-import type {
-  OptimisticId,
-  Id,
-  DestroyOptions,
-  SaveOptions
-} from './types'
+import apiClient from './apiClient'
+import deepmerge from 'deepmerge'
+
+import { OptimisticId, Id, DestroyOptions, SaveOptions } from './types'
 
 const dontMergeArrays = (_oldArray, newArray) => newArray
 
 export default class Model extends Base {
-  static defaultAttributes = {}
+  defaultAttributes = {}
 
   attributes: ObservableMap = observable.map()
   committedAttributes: ObservableMap = observable.map()
 
   optimisticId: OptimisticId = uniqueId('i_')
-  collection: ?Collection = null
+  collection: Collection<any> | null = null
```

Comment:
We maybe have a way to know how to parameterize that collection from this model.

Reply:
done 👍